### ISSUE TITLE
Relative image path in CSS

### ISSUE THREAD

**Issue by jericahuang** [Cheurina/prj-rev-bwfs-dasmoto#4]
You can also use relative image paths for your background-image settings in CSS!
https://github.com/Cheurina/prj-rev-bwfs-dasmoto/blob/master/Dasmoto/resources/css/style.css#L9

You're going to want to think about where in the file structure the image is located relative to the CSS file.

**Comment by Cheurina** [comment]
Yes, I got that but only after submitting the project while reading some comments on Slack
